### PR TITLE
Added unit tests for errors in `extraction.py`

### LINKED ISSUE #56: Add unit tests for testing errors
Unit tests should be added to ensure the raised errors throughout the functions work properly.

### PR DESCRIPTION
**Pull request by gabb222**
Fixes #56

### CODE REVIEW

File: tspng/extraction.py [hunk 1]
```diff
@@ -92,8 +92,8 @@ def extract_from_file(path: str, mime_type: str=MIME_TYPE) -> Dict:
     if not os.path.exists(path):
         raise Exception(f"{path} does not exist.")
     #check if file exists
-    if os.path.isdir(path):
-        raise Exception(f"{path} is to a directory.")
+    if not os.path.isfile(path):
+        raise Exception(f"{path} is not a file.")
```
Comment (volks73) [inline tspng/extraction.py:96]: Let's add "The" to the message for `The {path} is not a file.`

**Comment by gabb222** [comment]
@Theia-Scientific/developers ready to review

File: tspng/test_extraction.py [hunk 2]
```diff
@@ -22,6 +22,12 @@ def test_extract():
     test_data = extract_from_folder('tests/assets')
     assert list(test_data.keys())==['tests/assets/example_file_1.ts.png','tests/assets/example_file_2.ts.png']
 
+def test_extract_fails():
+    try:
```
Comment (volks73) [inline tspng/test_extraction.py:26]: This is not correct. The `with pytest.raises(TypeError):` structure should be used.

```python
def test_extract_fails():
    with pytest.raises(TypeError):
        extract("A string")
```

This style should be used for all of the "failing" tests.

**Review by volks73 — CHANGES_REQUESTED** [review]
The structure of the failing tests should use the `with` construct.

File: tspng/extraction.py (hunk 1 above)
Comment (gabb222) [reply tspng/extraction.py:96]: Change made as requested

File: tspng/test_extraction.py (hunk 2 above)
Comment (gabb222) [reply tspng/test_extraction.py:26]: Will make this change for those tests

**Comment by gabb222** [comment]
@Theia-Scientific/developers ready for review

File: tspng/test_extraction.py [hunk 3]
```diff
@@ -31,23 +37,44 @@ def test_extract_from_bytes():
...
     Tests the dictionary keys from an example TSPNG file path.
     '''
     test_data = extract_from_file('tests/assets/example_file_1.ts.png')
     assert list(test_data.keys())==['info','licenses','images','annotations','models','categories']
 
+def test_extract_from_file_fails():
+    #test path does not exist
+    with pytest.raises(Exception):
+        extract_from_file('Random/path.png')
+    #test file does not exist
```
Comment (volks73) [inline tspng/test_extraction.py:55]: Comment should be "test folder instead of file"

**Review by volks73 — APPROVED** [review]
Looks good.

One minor, optional change.

File: tspng/test_extraction.py (hunk 3 above)
Comment (gabb222) [reply tspng/test_extraction.py:55]: Made change as requested

**Pull request merged**